src/FsAutoComplete.Core/Consts.fs

```diff
@@ -0,0 +1,10 @@
+namespace FsAutoComplete.Core
+
+module ProjectLoader =
+  [<Literal>]
+  let ProduceReferenceAssembly = "ProduceReferenceAssembly"
+
+  let globalProperties =
+    [
+      // For tooling we don't want to use Reference Assemblies as this doesn't play well with type checking across projects
+      ProduceReferenceAssembly, "false" ]
```

Comment:
This is the fix for reference assemblies being on by default. I had to inline it in this PR to get accurate testing.

Reply:
makes sense :+1: